install/install.go

```diff
@@ -671,8 +672,8 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	}
 
 	if err := c.K8sClientset.RbacV1().Roles(o.Namespace).Delete(context.Background(), deployments.KubeArmorControllerLeaderElectionRoleName, metav1.DeleteOptions{}); err != nil {
-		if !strings.Contains(err.Error(), "not foundists") {
-			fmt.Print("Error while uninstalKubeArmor Controller Roleling \n")
+		if !strings.Contains(err.Error(), "not found") {
```

Comment:
not foundists! :-))

Reply:
`sed` can sedly cause these sometimes :P